src/olympia/devhub/templates/devhub/new-landing/components/my-addons.html

```diff
@@ -25,23 +25,23 @@ <h2>{{ _("Welcome Back") }}</h2>
       </div>
       <div>
         <p>
-          {% trans %}
-          Help users discover your extension by creating an appealing product page.
+          {% trans amo_link_open='<a href="https://addons.mozilla.org/">'|safe, amo_link_close='</a>'|safe %}
+          Want your extension to be featured on {{ amo_link_open }}addons.mozilla.org{{ amo_link_close }}?
+          Our featured extensions policies will tell you what you need to know.
           {% endtrans %}
         </p>
-        <a href="https://developer.mozilla.org/Add-ons/Listing" class="Button">
+        <a href="https://developer.mozilla.org/Add-ons/AMO/Policy/Featured" class="Button">
           {{ _('Learn How') }}
```

Comment:
```suggestion
          {{ _('Learn More') }}
```

@Passionatecricketer We need to change it to `Learn More` as well.